server/test/unit/server/async/test_tasks.py

```diff
@@ -1059,3 +1059,12 @@ def test_is_not_worker_is_scheduler(self):
 
     def test_is_not_worker_is_resource_mgr(self):
         self.assertEquals(tasks._is_worker("resource_manager@some.hostname"), False)
+
+
+class TestPulpTask(unittest.TestCase):
+
+    def check_task_type(self):
+        for task_name, task in app.celery.tasks.iteritems():
+            if task_name.startswith('pulp'):
+                if not isinstance(task, tasks.PulpTask):
+                    self.fail('task named %s must inherit from %s' % (task_name, tasks.PulpTask))
```

Comment:
This needs to be adjusted slightly to be.  `'task named %s must have %s as an ancestor'`